Documentation/ceph-cluster-crd.md

```diff
@@ -45,12 +45,88 @@ spec:
 ## PVC-based Cluster
 
 In a "PVC-based cluster", the Ceph persistent data is stored on volumes requested from a storage class of your choice.
-This type of cluster is recommended in a cloud environment where volumes can be dynamically created and also
+This type of cluster is recommended to use with CSI drivers which support dynamic volume provisioning and also in clusters where a local PV provisioner is available.
 in clusters where a local PV provisioner is available.
 
 > **NOTE**: Kubernetes version 1.13.0 or greater is required to provision OSDs on PVCs.
 
 ```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
```

Comment:
Please add this example yaml in the examples folder, for example named cluster-on-local-pvs.yaml. It's easier to use the examples if we have them in the example folder.